doc/build-netbsd.md

```diff
@@ -64,19 +64,26 @@ pkgin install db4
 ```
 
 #### GUI Dependencies
+###### Qt5
 
-Bitcoin Core includes a GUI built with the cross-platform Qt Framework. To compile the GUI, Qt 5 is required.
+Bitcoin Core includes a GUI built with the cross-platform Qt Framework. To compile the GUI, we need to install
+the necessary parts of Qt and libqrencode and pass `-DBUILD_GUI=ON`. Skip if you don't intend to use the GUI.
```

Comment:
tiny nit:

Should be ".. need to install the necessary parts of Qt, the libqrencode and pass `-DBUILD_GUI=ON`" (instead of "parts of Qt and libqrenconde and pass..").

Reply:
Thanks! Adjusted.